setup-server-os

```diff
@@ -28,6 +28,8 @@ def get_kernel_path():
     """
     if 'system-arch' in cfg.modules:
         return 'vmlinuz-linux'
+    if 'system-debian' in cfg.modules:
+        return 'vmlinuz-' + os.uname()[2]
```

Comment:
this is insufficient, unfortunately.
after a kernel update and `apt clean` the `setup-server-os` must be re-run or the system won't stiefel anymore.
we need to change the server script to list the /boot directory, automatically search for the newest kernel it can find, and boot that - for example by listing the dir, or by reading the grub config.